### PR TITLE
MRG: Fix ELP coords

### PR DESCRIPTION
**Pull request by Eric89GXL**
Closes #3755.

Improves on tests for coords by double-checking against values from commit 2fa35d4.

@jona-sassenhagen can you see if this fixes your problem?

### CODE REVIEW

**Comment by Eric89GXL** [comment]
(This is a within-cycle bugfix so no need for `whats_new` update.)

**Comment by jona-sassenhagen** [comment]
Will try, thanks Eric ...

**Comment by jona-sassenhagen** [comment]
Ok works for me:

Before:
unknown-125


After:
unknown-122

File: mne/channels/tests/test_montage.py [hunk 1]
```diff
@@ -40,68 +40,90 @@
...
+        elp='346\n'
+            'EEG\t      F3\t -62.027\t -50.053\t      85\n'
+            'EEG\t      Fz\t  45.608\t      90\t      85\n'
+            'EEG\t      F4\t   62.01\t  50.103\t      85\n'
+            'EEG\t      FCz\t   68.01\t  58.103\t      85\n',
+        hpts='eeg Fp1 -95.0 -3. -3.\n'
+             'eeg AF7 -1 -1 -3\n'
+             'eeg A3 -2 -2 2\n'
+             'eeg A 0 0 0',
+    )
```
Comment (jona-sassenhagen) [inline mne/channels/tests/test_montage.py]: Ha, I thought there was some purpose to it being formatted really ugly ... Looks much better this way.

**Comment by agramfort** [comment]
@jona-sassenhagen merge if you're happy

thx @Eric89GXL

**Comment by jona-sassenhagen** [comment]
I have pressed the big green button. Thanks Eric!

**Pull request merged**